src/coreclr/jit/emitarm64.cpp

```diff
@@ -13210,6 +13307,53 @@ void emitter::emitInsSve_R_R_R_I(instruction ins,
             fmt = IF_SVE_GZ_3A;
             break;
 
+        case INS_sve_luti2:
+            assert(insScalableOptsNone(sopt));
+            assert(isVectorRegister(reg1));
```

Comment:
Nit: Could you add the register/immediate comments (like `// mmmmm`, `// ddddd`, etc) here for consistency, please?

Reply:
I can, though lately (even some of my merged PRs) I've been excluding them from these paths as it adds a lot of redundancy.  I do keep them in `emitOutputInstr` however as that is where it is most relevant.